File: app/src/main/java/vn/tale/architecture/top_repos/TopReposComponent.java
package vn.tale.architecture.top_repos;

import dagger.Subcomponent;

/**
 * Created by Giang Nguyen on 1/14/17.
 */
@Subcomponent(modules = TopReposModule.class)
public interface TopReposComponent {

  void inject(TopReposActivity topReposActivity);
}

File: app/src/main/java/vn/tale/architecture/top_repos/TopReposComponent.kt
package vn.tale.architecture.top_repos

import dagger.Subcomponent

/**
 * Created by Giang Nguyen on 1/14/17.
 */
@Subcomponent(modules = arrayOf(TopReposModule::class))
interface TopReposComponent {

  fun inject(topReposFragment: TopReposFragment)
  fun inject(topReposActivity: TopReposActivity)
}
